app/models/post_action.rb

```diff
@@ -143,7 +143,7 @@ def self.count_per_day_for_type(post_action_type, opts = nil)
     result = unscoped.where(post_action_type_id: post_action_type)
     result = result.where('post_actions.created_at >= ?', opts[:start_date] || (opts[:since_days_ago] || 30).days.ago)
     result = result.where('post_actions.created_at <= ?', opts[:end_date]) if opts[:end_date]
-    result = result.joins(post: :topic).merge(Topic.in_category_and_categories(opts[:category_id])) if opts[:category_id]
+    result = result.joins(post: :topic).merge(Topic.in_category_and_subcategories(opts[:category_id])) if opts[:category_id]
```

Comment:
If this typo wasn't caught by the existing specs, perhaps we should add a spec for it?

Reply:
I don’t follow, this PR contains exactly this, a spec for likes report. It wasn’t caught before because there was no spec for this report.